HACKING.md

```diff
@@ -0,0 +1,27 @@
+⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
+
+# Hacking on the MCD
+
+1. Create a cluster (e.g. using [libvirt](https://github.com/openshift/installer/blob/master/Documentation/dev/libvirt-howto.md))
+2. Copy `/opt/tectonic/node-annotations.json` to `/etc/machine-config-daemon/` file on each node and pass it over this substitution:
```

Comment:
nit: drop "file" from this line?  Folks should be able to figure out that `/opt/tectonic/node-annotations.json` is a file and `/etc/machine-config-daemon/` is a directory on their own ;).